src/convertToObject.js

```diff
@@ -1,18 +1,23 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString
+    .split(';')
+    .forEach(element => {
+      const [property, value] = element.split(':');
+
+      if (property && value) {
+        const formatProperty = property
+          .replace('\n', '')
+          .trim();
+
+        result[formatProperty] = value.trim();
+      }
+    });
```

Comment:
Great realization. Please try to use a method that returns an object.
You can split code for more readability.

Reply:
I dont fully understand what i need to do
I have already return an object

Reply:
use `reduce` method instad `forEach`